style.css

```diff
@@ -95,3 +121,91 @@ button:focus-visible {
     background-color: #f9f9f9;
   }
 }
+
+.modal {
+    display: none;
+    /*overflow: auto;*/
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.5);
+}
+
+.modal_content {
+    text-align: left;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-30%, 0%);
+    width: 30%;
+    border-radius: 10px;
+    background-color: white;
+    padding: 10px;
+}
+
+.modal_buttons {
+    float: right;
+}
+
+.OK_button {
+    background-color: royalblue;
+    color: white;
+
+}
+
+.OK_button:hover {
+    /*  I am not able to make color beautiful ((((  */
```

Comment:
You got a royalblue color, which has a `#4169e1` code.
To understand it, you can double-click on color in Chrome DevTools:
<img width="431" alt="image" src="https://github.com/coolassassin/tooltip-and-modal/assets/17389477/dd7126ba-0642-4e37-a8bd-5b63edf29798">

Then you can open site like that:
http://hclwizard.org:3000/hclcolorpicker/

Here, click to `Hue-Chroma plane` and then you can use last bar to select lighter or darker color:
<img width="1135" alt="image" src="https://github.com/coolassassin/tooltip-and-modal/assets/17389477/491d9d06-bc26-4d0e-94ad-9ac0c42937a8">

I picked #5277ED as lighter and #2A5DD7 as darker variants. Both seem ok.

Colour theory is not trivial thing 🙃

And yeah, you also can ask ChatGPT for that:
<img width="709" alt="image" src="https://github.com/coolassassin/tooltip-and-modal/assets/17389477/b79e1765-47b9-44f2-a7ef-439b02e9dbda">